File: 3'rd Semester/assig/inCPP/q4.cpp
#include <iostream>
#include "Stack.cpp"

int main() {
    int r;
    Stack s = Stack(5);
    int num = 100;

    while (num > 0) {
        r = num % 2;
        s.push(r);
        num = num / 2;
    }

    while (!(s.isEmpty())) {
        cout << s.pop() << endl;
    }
}

File: 3'rd Semester/assig/q4.java

public class q4 {
    public static void main(String[] args) {
        Stack stack = new Stack(20);

        int num = 10;

        while (num > 0) {
            int r = num % 2;
            stack.push(r);
            num = num / 2;
        }

        while (!(stack.isEmpty())) {
            System.out.print(stack.pop());
        }
    }
}
